price-feeder/oracle/client/client.go

```diff
@@ -144,6 +145,7 @@ func (oc OracleClient) BroadcastTx(nextBlockHeight int64, timeoutHeight int64, m
 
 		resp, err := BroadcastTx(clientCtx, factory, msgs...)
 		if resp != nil && resp.Code != 0 {
+			telemetry.IncrCounter(1, "failure", "tx")
 			err = fmt.Errorf("invalid response code from tx")
```

Comment:
While we're at it, I forgot to mention in the other PR, let's add a `code` field to the error message.

Reply:
Updated these lines to : 
```go
telemetry.IncrCounter(1, "failure", "tx", "code")
err = fmt.Errorf("invalid response code from tx: %d", resp.Code)
```